Sources/GitHub/PullRequestAPI.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+extension PullRequest {
+	public final class API: GitHub.API {
+		public init(accessToken: String) {
+			super.init(authorization: .token(accessToken))
+		}
+
+		public func createPullRequest(
```

Comment:
<!--
  1 failure:  Function should h...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/GitHub/PullRequestAPI.swift;
  Line: 9;
-->
- :no_entry_sign: Function should have 5 parameters or less: it currently has 6 (`function_parameter_count`)